keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/binary/MaxVertex.java

```diff
@@ -11,7 +13,9 @@
      * @param left  one of the vertices to find the maximum of
      * @param right one of the vertices to find the maximum of
      */
-    public MaxVertex(DoubleVertex left, DoubleVertex right) {
+    @ExportVertexToPythonBindings
+    public MaxVertex(@LoadParentVertex(THEN_NAME) DoubleVertex left,
```

Comment:
I know I've made this wrap the `IF` and that's why we have the `THEN` and the `ELSE` name but it doesn't  make much sense for `Min` and `Max`. Can we change these names to `LEFT` and `RIGHT` to match the double binary op's??

Reply:
That won't work as the saves are annotated in the parent class.  I can put in a "pseudonym" final string - but it will still get saved as "then" and "else" due to the way the serialiser works in these cases.